watertap/examples/flowsheets/oaro/oaro.py

```diff
@@ -0,0 +1,480 @@
+###############################################################################
+# WaterTAP Copyright (c) 2021, The Regents of the University of California,
+# through Lawrence Berkeley National Laboratory, Oak Ridge National
+# Laboratory, National Renewable Energy Laboratory, and National Energy
+# Technology Laboratory (subject to receipt of any required approvals from
+# the U.S. Dept. of Energy). All rights reserved.
+#
+# Please see the files COPYRIGHT.md and LICENSE.md for full copyright and license
+# information, respectively. These files are also available online at the URL
+# "https://github.com/watertap-org/watertap/"
+#
+###############################################################################
+import os
+from pyomo.environ import (
+    ConcreteModel,
+    value,
+    Constraint,
+    Expression,
+    Objective,
+    Var,
+    Param,
+    NonNegativeReals,
+    TransformationFactory,
+    units as pyunits,
+    assert_optimal_termination,
+    check_optimal_termination,
+)
+from pyomo.network import Arc, SequentialDecomposition
+from idaes.core import FlowsheetBlock
+from idaes.core.solvers import get_solver
+from idaes.core.util.exceptions import InitializationError
+from idaes.core.util.model_statistics import degrees_of_freedom
+from idaes.core.util.initialization import (
+    solve_indexed_blocks,
+    propagate_state,
+)
+from idaes.models.unit_models import Mixer, Separator, Product, Feed
+from idaes.models.unit_models.mixer import MomentumMixingType
+from idaes.core import UnitModelCostingBlock
+import idaes.core.util.scaling as iscale
+import idaes.logger as idaeslog
+from idaes.core.util.misc import StrEnum
+
+import watertap.property_models.NaCl_prop_pack as props
+from watertap.unit_models.reverse_osmosis_0D import (
+    ReverseOsmosis0D,
+    ConcentrationPolarizationType,
+    MassTransferCoefficient,
+    PressureChangeType,
+)
+from watertap.unit_models.osmotically_assisted_reverse_osmosis_0D import (
+    OsmoticallyAssistedReverseOsmosis0D,
+    # ConcentrationPolarizationType,
+    MassTransferCoefficient,
+    PressureChangeType,
+)
+from watertap.unit_models.pressure_changer import Pump, EnergyRecoveryDevice
+from watertap.core.util.initialization import assert_degrees_of_freedom
+from watertap.costing import WaterTAPCosting
+
```

Comment:
Seems like some of these imports are unused and can be deleted.

Reply:
Yeah, I cleaned up redundant imports